manager-parent/manager-v2/src/test/java/com/redhat/service/smartevents/manager/v2/api/user/CloudProviderAPIV2Test.java

```diff
@@ -0,0 +1,104 @@
+package com.redhat.service.smartevents.manager.v2.api.user;
+
+import org.junit.jupiter.api.Test;
+
+import com.redhat.service.smartevents.infra.v2.api.V2APIConstants;
+import com.redhat.service.smartevents.manager.core.api.models.responses.CloudProviderListResponse;
+import com.redhat.service.smartevents.manager.core.api.models.responses.CloudProviderResponse;
+import com.redhat.service.smartevents.manager.core.api.models.responses.CloudRegionListResponse;
+import com.redhat.service.smartevents.manager.core.api.models.responses.CloudRegionResponse;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@QuarkusTest
+public class CloudProviderAPIV2Test {
```

Comment:
Yeah I see your point. Actually I duplicated the tests on purpose: given that V1 expects a particular payload and it should not change, this test ensures that if the `core` implementation changes the test will fail. It's a bit a defensive approach so that we don't change the `core` implementation breaking what is expected on `v1`. wdyt?

Reply:
Fair point. Not worth arguing about.